propagators/opentelemetry-propagator-instana/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './InstanaPropagator';
```

Comment:
Is it intended to export everything from InstanaPropagator.ts?
I guess it should be enough to only export `InstanaPropagator`.
It doesn't harm to export the header names but usually no user should tinker with them on their own.

Reply:
I don't care much either way. The other propagators also export `*` from their index.ts, so I went with that. I have reduced the export to the `InstanaPropagator` class now.